app/src/main/java/pl/droidevs/books/library/LibraryViewModel.java

```diff
@@ -0,0 +1,45 @@
+package pl.droidevs.books.library;
+
+import android.arch.lifecycle.LiveData;
+import android.arch.lifecycle.ViewModel;
+import android.os.AsyncTask;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+import pl.droidevs.books.entity.Book;
+import pl.droidevs.books.repository.BookRepository;
+
+public class LibraryViewModel extends ViewModel {
+
+    private BookRepository bookRepository;
+
+    @Inject
+    public LibraryViewModel(BookRepository bookRepository) {
+        this.bookRepository = bookRepository;
+        putTestData();
+    }
+
+    //TODO Remove
+    private void putTestData() {
+        new PopulateDbAsyncTask().execute("");
+    }
+
+    public LiveData<List<Book>> getBooks() {
+        return bookRepository.getBooks();
```

Comment:
Zobacz proszę skecję Transform LiveData na https://developer.android.com/topic/libraries/architecture/livedata.html

Reply:
Chodzi Ci konkretnie o to?: 
private LiveData<String> getPostalCode(String address) {
       // DON'T DO THIS
       return repository.getPostCode(address);
    }

Reply:
słuchaj, może Ty byś mógł to poprawić? 
Przyznam szczerze, że mogę mieć ograniczoną dostępność w weekend a wydaje mi się, że brak postawionej architektury jest blokerem dla pracy innych :)